examples/default/realign.json

```diff
@@ -0,0 +1,91 @@
+{
+    "@context": 
+      {
+        "bids" : "https://some/url/to/bids_context.jsonld",
+        "prov" : "http://www.w3.org/ns/prov#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "exp_dir" : {
+          "@id" : "exp_dir",
+          "prov:atLocation" : "$HOME/nidm-results-examples/spm-default/ds011/"
+        },
+        "startedAtTime" : {
+          "@id": "http://www.w3.org/ns/prov#startedAtTime",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "endedAtTime" : {
+          "@id": "http://www.w3.org/ns/prov#endedAtTime",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "wasGeneratedBy" : {
+          "@id": "http://www.w3.org/ns/prov#wasGeneratedBy",
+          "@type": "http://www.w3.org/2001/XMLSchema#Entity"
+        },
+        "wasAssociatedWith" : {
+          "@id": "http://www.w3.org/ns/prov#wasAssociatedWith",
+          "@type": "http://www.w3.org/2001/XMLSchema#Agent"
+        },
+        "wasInformedBy" : {
+          "@id": "http://www.w3.org/ns/prov#wasInformedBy",
+          "@type": "http://www.w3.org/2001/XMLSchema#Activity"
+        },
+        "used" : {
+          "@id": "http://www.w3.org/ns/prov#used",
+          "@type": "http://www.w3.org/2001/XMLSchema#Entity"
+        },
+        "quality" : {
+          "@id": "TODO",
+          "@type": "xsd:float"
+        },
+        "sep" : {
+          "@id" : "TODO",
+          "@type" : "xsd:int" 
+        },
+        "which" : {
+          "@id" : "TODO",
+          "@type" : "xsd:list"
+        }
+      },
+    "@id": "http://example.org/ds00000X",
+    "generatedAt": "2020-03-10T10:00:00",
+    "wasGeneratedBy": {
+        "@id": "INRIA",
+        "@type": "Project",
+        "startedAt": "2016-09-01T10:00:00",
+        "wasAssociatedWith": { "@id": "NIH",
+                               "@type": "Organization",
+                               "hadRole": "Funding"
+                             }
+      },
+    "@graph": [
+      {
+        "@id" : "realign",
+        "@type" : "prov:Activity",
+        "wasAssociatedWith" : "estwrite",
+        "startedAtTime" : "10/10/2020 00:00:00",
+        "endedAtTime" : "10/10/2020 01:00:00",
+        "used" : ":exp_dir/PREPROCESSING/ANATOMICAL/*",
+        "eoptions" : {
```

Comment:
Replace by something like:
```
"attributes" : [
      "eoptions.quality": 0.9,
      "roptions.which": [0,1],
      "roptions.interp": 4,
]
```

Reply:
<--- This is not correct we should think about how to have "eoptions.quality" as string not term to avoid having to create a term for each new attribute.

Reply:
`"attributes": ["eoption.quality=0.9", "roptions.which=[0,1]"]`

Reply:
`"attributes": [["eoption.quality", 0.9], ["roptions.which", [0,1]]]`